### PR TITLE
Upgrade @kiwicom/relay to latest version

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: apps/core/scenes/results/Results.js [hunk 1]
```diff
@@ -103,6 +79,14 @@ export default class Results extends React.Component<Props> {
       DateFNS.format(DateFNS.parseISO(date), ORDINAL_DAY_MONTH_FORMAT);
 
     const tripType: TripTypes = returnDateFrom ? 'Return' : 'OneWay';
+    const QueryComponent =
+      tripType === 'Return' ? ReturnResultsQuery : OneWayResultQuery;
```
Comment (aalises) [inline apps/core/scenes/results/Results.js]: Thanks for the refactoring! Cleaner this way. One thing, maybe we should rename `ReturnResultsQuery` and `OneWayResultQuery` to be either both `<Type>ResultsQuery` or `<Type>ResultQuery` for consistency.

Comment (tbergq) [reply apps/core/scenes/results/Results.js]: Not sure I understand exactly what you want to change, could you please elaborate?

Comment (aalises) [reply apps/core/scenes/results/Results.js]: It's just a naming convention, Return **Results**Query and OneWay**Result**Query, maybe putting both to use either Result or Results for consistency would be nice 💃

Comment (tbergq) [reply apps/core/scenes/results/Results.js]: Yes, you are right, I didn't catch that. I will name them both with plural

**Comment by tbergq** [comment]
@JoseDuSV @aalises 🚀

**Pull request merged**